docs/theme.config.tsx

```diff
@@ -28,8 +28,7 @@ const config: DocsThemeConfig = {
   feedback: {
     content: null,
   },
-  docsRepositoryBase:
-    'https://github.com/TurtIeSocks/Koji/tree/main/docs/pages',
+  docsRepositoryBase: 'https://github.com/TurtIeSocks/Koji/tree/main/docs',
```

Comment:
```suggestion
  docsRepositoryBase: 'https://github.com/TurtIeSocks/Koji/edit/main/docs',
```
This should send users straight to the editor view